### PR TITLE
Adding hasAttribute to mirror hasClass.

### PR DESCRIPTION
**Pull request by jasonrose**
This is to support the tests I'm writing for https://github.com/AmpersandJS/ampersand-dom-bindings/issues/33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ dom.switchClass(el, 'oldclass', 'newclass');
 // makes sure attribute (with no content) is added
 // if exists it will be cleared of content
 dom.addAttribute(el, 'checked');
+dom.hasAttribute(el, 'checked'); // => true
```
Comment (wraithgar) [inline README.md]: Maybe add a comment before this explaining what it is and that it returns a boolean?

**Comment by wraithgar** [comment]
+1 visual. a good addition with tests

File: README.md (hunk 1 above)
Comment (jasonrose) [inline README.md]: Yeah, I'll update.  I just copied the snippet from `hasClass` above it.

**Comment by lukekarrys** [comment]
+1

**Comment by lukekarrys** [comment]
Published as `v1.3.0`

**Pull request merged**